ci/provisioning/post_provision_config_nodes_EL_8.sh

```diff
@@ -65,31 +38,44 @@ distro_custom() {
         cat <<EOF >> /etc/yum.repos.d/daos_ci-rocky8-artifactory.repo
```

Comment:
Shouldn't we just add these to the repo_files: *.repo files for each distro/version?
Then we just make sure that the most up to date repo files are installed as part of post provisioning until we get the snapshot restore script to do that.

Reply:
Yes.  And we will.  But the existing images don't have them and since one of the goals of this PR is to function in the absence or failure of artifactory, including the ability to fetch repo files, it has to work with the repos defined on the existing images.